_includes/header.html

```diff
@@ -73,16 +73,18 @@
 
             <div class="item active">
               <div class="container">
-                <div class="jumbotron col-md-12">
+                <div class="jumbotron">
                   <h1>We are Code for <br/> Hampton Roads</h1>
                   <div class="row">
                   <p class="lead col-md-10 col-md-offset-1">
                     Come join developers, designers, data geeks, leaders, and idea-makers who volunteer to help local government and civic orgs adopt open web technologies.
                   </p>
-                  </div>
+                  </div class="row">
```

Comment:
It's not necessary to set a class in the closing tag.  You can remove `class="row"`.